kernel/smp.c

```diff
@@ -48,3 +50,56 @@ void _smp_global_unlock(unsigned int key)
 	recursive_count = 0;
 	k_spin_unlock(&global_spinlock, sk);
 }
+
+extern k_thread_stack_t _interrupt_stack1[];
```

Comment:
I'd really like to see these interrupt stacks and idle thread objects, etc. expressed as an array sized by some Kconfig, rather than hard-coded in this way.
We can punt to a future enhancement if needed.